src/App.js

```diff
@@ -236,7 +236,7 @@ function App() {
       );
       ReactDOM.render(fileBtn, document.getElementById("fileBtn"));
     } else {
-      history.push('/?key='+keys[0]);
+      history.push('/keys/?key='+keys[0]);
```

Comment:
/?key will also work if we wrap the routing in the index.js in Switch

Reply:
Yes.. i completely forgot to wrap it. Will send a new PR

Reply:
Done

Reply:
Should be 
history.push('/?key='+keys[0]);